tests/jobs/test_scheduler_job.py

```diff
@@ -842,21 +842,12 @@ def test_find_dags_to_run_includes_subdags(self):
         print(self.dagbag.dag_folder)
         self.assertGreater(len(dag.subdags), 0)
         dag_file_processor = DagFileProcessor(dag_ids=[], log=mock.MagicMock())
-        dags = dag_file_processor._find_dags_to_process(self.dagbag.dags.values(), paused_dag_ids=())
+        dags = dag_file_processor._find_dags_to_process(self.dagbag.dags.values())
 
         self.assertIn(dag, dags)
         for subdag in dag.subdags:
             self.assertIn(subdag, dags)
 
-    def test_find_dags_to_run_skip_paused_dags(self):
-        dagbag = DagBag(include_examples=False)
-
-        dag = dagbag.get_dag('test_subdag_operator')
```

Comment:
Although the interface has changed, we do want to check it still skips over paused dags -- do we have a test that covers that still?

Reply:
I thought that @kaxil had already added such a test, but I will check it and add it if it is not there yet.

Reply:
We refactored a method: https://github.com/apache/airflow/pull/7587 and added a test to check the response.

and we relied on the 2 existing test, check the comment here: https://github.com/apache/airflow/pull/7587#issuecomment-592527753

This test was one of them, we don't need to delete this test, do we?

Reply:
We need to remove it because we are now checking for active DAGs in the processs_file method. Here is commit: https://github.com/apache/airflow/pull/7674/commits/708872191af53a81cf74a9be3ea2309db0b8a748
I added new tests:
https://github.com/apache/airflow/pull/7674/commits/7232a990d7648052d6e18ddb6ba38342ec0705c7